src/org/zaproxy/zap/extension/ascanrules/TestSQLInjection.java

```diff
@@ -1472,6 +1467,46 @@ protected String stripOff(String body, String pattern) {
 	}
 
 	/**
+	 * Replace body by stripping of pattern string. The URLencoded pattern will
+	 * also be stripped off. The URL decoded pattern will not be stripped off,
+	 * as this is not necessary of rour purposes, and causes issues when
+	 * attempting to decode parameter values such as '%' (a single percent
+	 * character) This is mainly used for stripping off a testing string in HTTP
+	 * response for comparison against the original response. Reference:
+	 * TestInjectionSQL
+	 *
+	 * @param body
+	 * @param pattern
+	 * @return
```

Comment:
There's no point including these 3 or 4 lines if they're not going to be defined/documented.